src/store/index.ts

```diff
@@ -922,19 +905,63 @@ ircClient.on("CAP LS", ({ serverId, cliCaps }) => {
   console.log(`Server ${serverId} supports capabilities: ${cliCaps}`);
 });
 
-ircClient.on("CAP_ACKNOWLEDGED", ({ serverId, capabilities }) => {
-  if (capabilities === "sasl") {
+ircClient.on("CAP_ACKNOWLEDGED", ({ serverId, key, capabilities }) => {
```

Comment:
I think this should be also encapsulated in the ircClient only, instead of being exposed here, but coudln't figure it out right now